less/pages/encrypt/components.less

```diff
@@ -563,6 +563,11 @@ select{
   }
 }
 
+.privacy-notice a {
```

Comment:
I had to move this anchor tag into a localized string, so localizes can possibly move the anchor around, which means I also needed to load the string via a react-intl function to localize text with the potential to have html, which means the markup changed, and the previous css specificity no longer worked. I'll reference this message later when I interact with changes that relate to this.